.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
```

Comment:
> Issue templates and their configuration file must be in a folder called .github/ISSUE_TEMPLATE. All other supported files must be in the root of the repository. For more information, see "Creating new files."

The [linked docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) mention that _issue templates_ specifically must be in a .github dir. It seems strange to me that this would not apply to PR templates; we probably need to merge this and test to figure out exactly where each file must be.